projects/plugins/jetpack/_inc/jetpack-modules.models.js

```diff
@@ -61,6 +59,7 @@ window.jetpackModules.models = ( function( window, $, _, Backbone ) {
 			// Sort unavailable modules to the end if the user is running in local mode.
 			groups = _.groupBy( items, 'available' );
 			if ( _.has( groups, 'false' ) ) {
+				// eslint-disable-next-line dot-notation -- Rule is confused.
 				items = [].concat( groups[ true ], groups[ false ] );
```

Comment:
More specifically it wants to do like `group.true`, which is equivalent to `group[ 'true' ]` rather than `group[ true ]`.

Reply:
I'm not quite sure why anyone would create a bool key...perhaps it was a mistake?

That said, it seems JS implicitly converts a bool key to a string, so I think it'd be safe to change:

```
asdf = {true:1}
asdf['true'] // 1
asdf.true // 1
```

Reply:
Note that the first line should be `asdf = { [true]: 1 }` to use boolean `true` as the key rather than the string `'true'`.

But you're right, JS does stringify `true` as a key to `'true'` rather than say `1` as I had thought it would.
```suggestion
				items = [].concat( groups.true, groups.false );
```